alpine/entrypoint/post_start.sh

```diff
@@ -5,47 +5,13 @@ do
    [[ "${LOGLINE}" == *"Binding bundle: [landing-page"* ]] && pkill -P $$ tail
 done
 
-if [ -n "$INSTALL_FEATURES" ]; then
-  if [[ $INSTALL_FEATURES == *";"* ]]; then
```

Comment:
I'm thinking it might be valuable to leave the `feature:install` and `feature:uninstall` cases in here as well for non ConfigureCommand scenarios. (like if we ever wanted to use a regular karaf container for something for some reason)